docs/assemblies/proc_deploying-in-disconnected-environments.adoc

```diff
@@ -0,0 +1,48 @@
+[id="proc_deploying-in-disconnected-environments"]
+= Deploying OpenStack in a disconnected environment
+
+[role="_abstract"]
+== Process
+Deploying in disconnected environments can be achieved largely by following the OpenShift documentation for mirroring OLM Operators: https://docs.openshift.com/container-platform/4.17/disconnected/mirroring/installing-mirroring-installation-images.html#olm-mirror-catalog_installing-mirroring-installation-images
+
+== Technical Overview
+The `openstack-operator` contains a list of related images that will ensure all required images for the deployment are mirrored following the above OpenShift process. Once images are mirrored, the `ImageContentSourcePolicy` custom resource (CR) is created. This process results in a `MachineConfig` called `99-master-genereted-registries` being updated in the cluster. The `99-master-generated-registries` `MachineConfig` contains a `registries.conf` file that is applied to all of the OpenShift nodes in the cluster.
+
+In order to integrate cleanly with this process. The dataplane controller will check for the existence of an `ImageContentSourcePolicy`. If one is found, we will read the `registries.conf` file from the `99-master-generated-registries` `MachineConfig`. The dataplane controller will then set two variables in the Ansible inventory for the nodes.
+
+[,yaml]
+----
+edpm_podman_disconnected_ocp
```

Comment:
Would not mentioning these ansible variables that are for internal use confuse the users? Though the section heading is `Technical Overview` i.e how it internally works, we should probably clearly mention that user does not have to do anything from OSP pov once OCP operataor catalog mirroring is done, right?